src/modules/News/News.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import classNames from 'classnames/bind';
+
+import { Paper } from '@components/Paper';
+
+import styles from './News.module.css';
+import { NewsCarousel } from './components/NewsCarousel';
+import { locale } from './locale';
+
+const cn = classNames.bind(styles);
+
+const images = [
```

Comment:
let's move it to mock in the next PR

Reply:
mock server and consume it from mock response?

Reply:
> mock server and consume it from mock response?

yes